easybuild/easyconfigs/s/SSW/SSW-1.1_build-cpp-lib.patch

```diff
@@ -0,0 +1,71 @@
+diff -Nru SSW.orig/src/Makefile SSW/src/Makefile
```

Comment:
Could you add a short description of what the patch does, and preferably who wrote it (or a reference to a PR/issue)  (same for all the patches)